specificatie/openapi.yaml

```diff
@@ -648,6 +810,174 @@ components:
+          properties:
+            status:
+              description: Een lijst met enumeratie waarden waar gezochte panden wel of niet aan moeten voldoen.
+              $ref: '#/components/schemas/PandStatusZoekObject'
+            geconstateerd:
+              description: Een indicatie of er naar geconstaeerde resources gezocht moet worden.
+              type: boolean
+            bouwjaar:
+              description: Een minimale en/of maximale waarde van het bouwjaar van paden waarnaar gezocht moet worden. 
+              $ref: '#/components/schemas/NumberOperator'
+
+    PandStatusZoekObject:
+      type: object
+      required:
+        - waarden
+      properties:
+        operator:
+          $ref: '#/components/schemas/EnumOperator'
+        waarden:
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/StatusPand_enum'
+
+    NumberOperator:
+      description: >-
+        "Een generieke aanduiding voor de min en max waarde van een attribuut van een object.
+        Als alleen een min waarde wordt opgegeven, dan wordt er gezocht naar waarden die groter dan of gelijk zijn aan de min waarde.
+        Als alleen een max waarde wordt opgegeven, dan wordt er gezocht naar waarden die kleiner dan of gelijk zijn aan de max waarde.
+        Als zowel min en max waarde worden opgegeven waarbij min < max dan wordt er gezocht naar waarden die groter dan of gelijk zijn aan min en kleiner dan of gelijk aan max, oftewel waarden die tussen min en max liggen.
+        Als zowel min en max waarde worden opgegeven waarbij min > max dan wordt er gezocht naar waarden die groter dan of gelijk zijn aan min en kleiner dan of gelijk aan max, oftewel waarden die niet tussen min en max liggen." 
+      type: object
+      properties:
+        min:
+          type: number
+        max:
+          type: number
+
+    ArrayOperator:
+      type: string
+      description: >-
+        "Een lijst operatie, de volgende operaties worden ondersteund:
+        - and   : de gezochte resource property moet alle waarden uit de lijst bevatten
+        - or    : de gezochte resource property moet één of meer waarden uit de lijst bevatten
+        - not   : de gezochte resource property bevat geen van de waarde uit de lijst"
+      enum:
+        - and
+        - or
+        - not
+      default: or
+
+    EnumOperator:
+      type: string
+      description: >-
+        "Een enumeratie operatie, de volgende operaties worden ondersteund:
+        - or    : de gezochte resource property moet een waarde hebben die in de lijst met waarden voorkomt 
+        - not   : de gezcohte resource property moet een waarde hebben die niet in de lijst met waarden voorkomt"
+      enum:
+        - or 
+        - not
+      default: or
+
     Adres:
       type: object
       description: "Eigenschappen van het adres die kunnen worden hergebruikt in andere API's waarin adresgegevens worden opgenomen. "
```

Comment:
"Eigenschappen van het adres die kunnen worden hergebruikt in andere API's met adresgegevens.